jax/lib/__init__.py

```diff
@@ -38,20 +38,14 @@
 def _check_jaxlib_version():
   if version < _minimum_jaxlib_version:
     msg = 'jaxlib is version {}, but this version of jax requires version {}.'
-
-    if version == (0, 1, 23):
```

Comment:
This one is not yet obsolete, I think. This is the error you get if you `pip install jax jaxlib` and your `pip` is too old and doesn't know about `manylinux2010`.

We could however fix this by deleting all wheels <= 0.1.23 from pypi.

Reply:
Good catch - I reverted this part.